boilerplate/flyte/flyte_golang_compile/Readme.rst

```diff
@@ -0,0 +1,16 @@
+Flyte Golang Compile
```

Comment:
We do not need the Golang stuff in flytekit (it's a python repo)... did you get these files by running `make boilerplate_update` or did you copy paste them? there should be a boilerplate/config file or something that list the directories to pull from boilerplate... and for flytekit, it shouldn't include the golang stuff

Reply:
I didn't include them in update.cfg...They just copied from update boilerplate